app/views/layouts/application.html.erb

```diff
@@ -27,11 +27,18 @@
     <a href="/" class="right-link" >Home</a>
     <%= image_tag current_user.avatar_url, size: User::AVATAR_SIZES[:small], class: "right-link" if current_user.avatar? %>
     <span class="right-link username"><%= current_user.name %></span><span class="right-link">Signed in as: </span>
+    <%= link_to new_post_path, class: "new-posts" do %>
+      New post
+    <% end %>
     <% else %>
     <%= link_to 'Sign in', sign_in_path %>
     <% end %>
+
   </nav>
-  <div class="header-buffer"></div>
+
+
+  <div class="header-buffer">
+  </div>
```

Comment:
I'd like to see this diff be just lines 30-32 — the whitespace and formatting changes seem to add churn without benefit.